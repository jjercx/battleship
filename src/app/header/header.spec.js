import React from "react";
import Header from "./header";
import renderer from "react-test-renderer";

describe("<Header/>", () => {
  it("matches snapshot", () => {
    const tree = renderer.create(<Header />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
