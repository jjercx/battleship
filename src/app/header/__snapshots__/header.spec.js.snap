// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> matches snapshot 1`] = `
<div
  className="sc-AxjAm zdmwr"
>
  <span
    className="sc-AxirZ bFLejp"
  >
    BATTLESHIP
  </span>
</div>
`;
